src/components/cards/NetworkReportCard.vue

```diff
@@ -25,6 +25,40 @@
               <div class="col-md-8">
                 <report-description :description="network.description" :maxLength="500"></report-description>
                 <report-details-list :reportDetails="detailsContent"></report-details-list>
+                <b-tabs v-if="collections === undefined || biobanks === undefined || collectionsAvailable || biobanksAvailable">
```

Comment:
Why did you put the `===undefined` there instead of `!collections` and `!biobanks`?